apps/common/js/components/leap-hands-view.jsx

```diff
@@ -56,19 +76,22 @@ export default class LeapHandsView extends React.Component {
     camera.position.fromArray([0, 500, 400]);
     camera.lookAt(new THREE.Vector3(0, 0, 0));
     scene.add(camera);
-    return {scene, camera};
+    return { scene, camera };
   }
 
   render() {
-    const { width, height } = this.props;
+    let width = this.width;
+    let height = this.height;
+    if (this.props.width != null){
+      width = this.props.width;
+      height = this.props.height;
+    }
```

Comment:
Hm, I think that changes to this file might have broken previous logic here (that probably was unclear, sorry for that!).

The previous idea was that you could set width and height using React props and CSS (e.g. 100%, 500px, 20em, etc.). `this.width` and `this.height` methods/getters were only used internally to get final dimensions in pixels and setup Three.JS renderer that expects value in pixels.

So, the order was:
- apply CSS styles to container using props
- get final dimensions in pixels (clientHeight and clientWidth)
- apply them to the Three.JS renderer

Now, I'm not sure how this is working. Also, not sure why we sometimes return height instead of width or width instead of height?

I think it might have broken other pages a bit:
![screen shot 2016-08-12 at 11 46 05](https://cloud.githubusercontent.com/assets/767857/17619263/7ab3e40c-6084-11e6-9880-a80406a93e62.png)
Note that hands view takes only ~50% of the model and it's on the left side. It used to cover the whole model.